src/test/java/RequestTest.java

```diff
@@ -16,14 +17,14 @@
 
 
 
-    @Test
-    public void inputSentToClient () throws IOException {
-        ByteArrayOutputStream mockOutput = new ByteArrayOutputStream();
-
-        sendResponse(mockOutput);
-
-        assertArrayEquals("status: 200 OK\n".getBytes(), mockOutput.toByteArray());
-    }
+//    @Test
+//    public void inputSentToClient () throws IOException {
+//        ByteArrayOutputStream mockOutput = new ByteArrayOutputStream();
+//
+//        sendResponse(mockOutput);
+//
+//        assertArrayEquals("status: 200 OK\n".getBytes(), mockOutput.toByteArray());
+//    }
 
     @Test
     public void RequestDataIsRead () throws IOException {
```

Comment:
Is this test related to the `Request` class? If not let's move it, or delete if it's no longer needed

Reply:
I find this test useful, but it's not correctly working atm due to some changes in what you are printing and lack of new line in `mockInput` . I've changed it this way to make it work: 

```ruby
    @Test
    public void RequestDataIsRead () throws IOException {
        ByteArrayInputStream mockInput = new ByteArrayInputStream("done\n somethig To parse".getBytes());
        ByteArrayOutputStream output = new ByteArrayOutputStream();
        System.setOut(new PrintStream(output));

        requestReader(mockInput);

        assertEquals("http.Server received : done\noutput read", output.toString().trim());
    }
```

Reply:
I had to change it slightly but it works now thank you 😄